MAUI/PDF-Viewer/Signature.md

```diff
@@ -0,0 +1,116 @@
+platform: MAUI
+control: SfPdfViewer
+documentation: ug
+---
+
+# Electronic Signature in .NET MAUI PDF Viewer (SfPdfViewer)
+
+The electronic signature feature of [SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html) allows you to add, remove signature in the PDF document. This section will go through the various types and functions available in PDF Viewer for working with signature.
+
+## Types of electronic signature
+
+The following signature types are now available in the PDF Viewer.
+
+1.	Handwritten signature.
+2.	Text signature.
+3.  Image signature.
+
+## Add signature by UI interaction
+
+This section will go through how to add signature by UI interaction to a PDF document.
+
+You can add signature to a PDF document with UI interaction using signature dialog. The following steps explains how to add signature on a PDF.
+
+	1. Set the AnnotationMode property of the SfPdfViewer to `Signature`. It activates the signature mode on the control and signature dialog box will be open.
+	2. create a signature in any one of the types
+	3. You can customize the color of the signature for handwritten and text signature types
+	4. Tap on the page to add the created signatures.
+	5. Once you have done, set the AnnotationMode to None. It will disable the signature mode and save the added signatures to the PDF page.
+	6. You can later move, resize, or delete the signature.
+	
+The following code explains how to enable the `Signature` annotation mode.
+
+{% tabs %}
+{% highlight c# %}
+// Enable or activate the signature mode.
+void EnableSignatureMode()
+{
+    // Set the annotation mode to signature using the `SfPdfViewer` instance.
+    PdfViewer.AnnotationMode = AnnotationMode.Signature;
+}
+{% endhighlight %}
+{% endtabs %}
+
+Similarly, refer to following code to disable the signature mode.
+
+{% tabs %}
+{% highlight c# %}
+// Disable or deactivate the ink drawing mode.
+void DisableSignatureMode()
+{
+    // Set the annotation mode to none using the `SfPdfViewer` instance.
+    PdfViewer.AnnotationMode = AnnotationMode.None;
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Add signature programmatically
+
+This section will go through how to add signature programmatically to a PDF document.
+
+Note: Currently, there is no support for add a text signature programmatically.
+
+### Add handwritten signature 
+
+To add a handwritten signature, You can create and add an ink annotation to a PDF document programmatically using the AddAnnotation method of the SfPdfViewer. Additionally in the case of signature you should set the `IsSignature` API of the ink annotation true. The following example explains how to create an ink annotation and add it as a signature to the first page of a PDF document.
+
+{% tabs %}
+{% highlight c# %}
+int pageNumber = 1;
+    
+// Provide the points collection to draw a stroke. Here a single stroke is created.
```

Comment:
// Provide the points collection to draw a stroke. Here, a single stroke is created.